variables.tf

```diff
@@ -693,6 +711,12 @@ variable "bamboo_db_iops" {
   type        = number
 }
 
+variable "bamboo_db_name" {
+  description = "The default DB name of the DB instance."
+  default     = null
+  type        = string
+}
+
```

Comment:
Do we need this for Bamboo? I think we use different approach rather than snapshot for Bamboo.

Reply:
But on the other hand, is good to be consistent.

Reply:
This is optional but I added to keep it consistent